### ISSUE TITLE
Lighthouse Report - General Accessibility

### ISSUE THREAD

**Issue by BrentM2005** [iraana/Capstone-Project#86]
The lighthouse report is an open-source, automated tool to help you improve the quality of web pages.  It is available on Google Chrome.  I would like to aim for 100% accessibility on the light house report for every page.  Here are the changes needed:

- Change College Administration, Emergency Situations, Access, Correction, Deletion, Data Portability, Opt-out, and Restriction to `<p>` instead of `<h4>` (PrivacyPolicyPage.tsx)
- Change Order Limits, Order Deadlines, Reservations to `<p>` instead of `<h4>` (ToSPage.tsx)
- Change Pickup Location, Phone, Service Hours to `<p>` instead of `<h3>` (Contact.tsx)
- Add a label to the agree checkbox (SignUp.tsx)
- Select a dish combo box needs a label (Reviews.tsx)
- Change button headings to `<p>` (AdminHome.tsx)
- Category combo box needs a label (AddDish.tsx)
- Remove aria-hidden (AddToGallery.tsx)
- Role combo box needs a label (ManageUsers.tsx / ManageAdmins.tsx)
- Detailed data report should be a `<p>` (Analytics.tsx)
- Menu combo box needs a label (PendingOrders.tsx / ArchivedOrders.tsx / CancelledOrders.tsx)
- Menu date selector needs a label (AddMenu.tsx)
- Select Date to Edit needs a label (EditMenu.tsx)
- Date selectors for date range don't have labels (AdminReviews.tsx)
- Navbar hamburger button and cart need labels (Navbar.tsx) (mobile specific)
- Every page needs a main tag
